test/SparqlAntlrParserTest.cpp

```diff
@@ -855,3 +855,102 @@ TEST(SparqlParser, GroupClause) {
                              IsExpressionGroupKey("COUNT(?baz)"));
   }
 }
+
+TEST(SparqlParser, propertyPaths) {
+  using Op = PropertyPath::Operation;
+  // Test all the base cases.
+  {
+    // "a" is a special case. It is a valid PropertyPath.
+    // It is short for "<http://www.w3.org/1999/02/22-rdf-syntax-ns#type>".
+    expectCompleteParse(
+        parseVerbPathOrSimple("a", {}),
+        IsPropertyPath(PropertyPath(
+            Op::IRI, 0, "<http://www.w3.org/1999/02/22-rdf-syntax-ns#type>",
+            {})));
+  }
```

Comment:
If everything is created temporarily in the call to `expectCompleteParse` then the `{}` around the block are completely redundant an can be removed.